sdk/monitor/Azure.ResourceManager.Monitor/api/Azure.ResourceManager.Monitor.netstandard2.0.cs

```diff
@@ -987,6 +1039,14 @@ public partial class AutoscaleSettingPredicativeResult
         public Azure.Core.ResourceIdentifier TargetResourceId { get { throw null; } }
         public string Timespan { get { throw null; } }
     }
+    public partial class CommonPrivateEndpointConnectionV4 : Azure.ResourceManager.Models.ResourceData
```

Comment:
This is not a good name - could we rename it to something like MonitorPrivateEndpointConnection?

Reply:
Updated to MonitorWorkspacePrivateEndpointConnection along with other PrivateEndpoint resources from v4/privatelinks.json